app/src/main/sentences/sl/telephone.dslf

```diff
@@ -0,0 +1,3 @@
+telephone: high
+(pokliči?)|(pozvoni)|telefoniraj|kontaktiraj .who.;
```

Comment:
```suggestion
pokliči|pozvoni|telefoniraj|kontaktiraj .who.;
```

Also here: with the `?` this would match everything. Also, you can probably combine the two sentences into a single one